tools/manual_typings/seed/cssnano.d.ts

```diff
@@ -3,6 +3,8 @@ declare module 'cssnano' {
   interface IOptions {
     discardComments: {
       removeAll: boolean;
+      discardUnused:boolean;
+      zindex:boolean;
     };
```

Comment:
This does'nt match the option you have in the task file. And (of course) it doesn't build (tsc error).
Can you please fix ?